sonaranalyzer-dotnet/src/SonarAnalyzer.VisualBasic/Rules/PropertiesAccessCorrectField.cs

```diff
@@ -68,6 +76,22 @@ public sealed class PropertiesAccessCorrectField : PropertiesAccessCorrectFieldB
             return null;
         }
 
+        private static FieldData? ExtractFieldFromRefArgument(ArgumentSyntax argument, Compilation compilation)
+        {
+            var semanticModel = compilation.GetSemanticModel(argument.SyntaxTree);
+            if (semanticModel != null && argument.Parent is ArgumentListSyntax argList)
+            {
+                var argumentIndex = argList.Arguments.IndexOf(argument);
+                if (semanticModel.GetSymbolInfo(argList.Parent).Symbol is IMethodSymbol methodSymbol &&
+                    argumentIndex < methodSymbol?.Parameters.Length &&
+                    methodSymbol?.Parameters[argumentIndex]?.RefKind == RefKind.Ref)
```

Comment:
What about _out_ parameters? Looks like an _out_ parameter would also update the field.
Either way, there should be a test case for it.